tests/operators/test_save_file.py

```diff
@@ -24,10 +24,11 @@
 import astro.dataframe as adf
 import astro.sql as aql
 from astro.constants import SUPPORTED_DATABASES, SUPPORTED_FILE_TYPES, Database
+from astro.files import File
 from astro.settings import SCHEMA
 
 # Import Operator
-from astro.sql.operators.agnostic_save_file import save_file
+from astro.sql.operators.save_file import save_file
```

Comment:
Do we have a test case covering the scenario: https://github.com/astronomer/astro-sdk/issues/365?

Reply:
I'll address this in a different PR, it's already too big.

Reply:
I'll add this.